packages/front-end/components/Teams/AddMembersModal.tsx

```diff
@@ -0,0 +1,73 @@
+import { TeamInterface } from "back-end/types/team";
+import { useForm } from "react-hook-form";
+import useApi from "@/hooks/useApi";
+import { useAuth } from "@/services/auth";
+import { useUser } from "@/services/UserContext";
+import Modal from "../Modal";
+import MultiSelectField from "../Forms/MultiSelectField";
+
+export const AddMembersModal = ({
+  teamId,
+  open,
+  onClose,
+}: {
+  teamId: string;
+  open: boolean;
+  onClose: () => void;
+}) => {
+  const { data, mutate } = useApi<{
+    team: TeamInterface;
+  }>(`/teams/${teamId}`);
+
+  const form = useForm<{
+    members: string[];
+  }>({
+    defaultValues: {
+      members: data?.team.members?.map((m) => m.id) || [],
```

Comment:
This will always be an empty array.  `data` is not defined on initial load and I believe useForm ignores changes to `defaultValues`.

I think we can get rid of the `useApi` call completely here, right?  From what I can tell, this modal is about adding new users, so we don't really care who the existing users are and should always set this to an empty string.